core/src/main/java/org/jruby/runtime/scope/ManyVarsDynamicScope.java

```diff
@@ -43,6 +53,7 @@ private void allocate() {
     }
     
     public DynamicScope cloneScope() {
```

Comment:
We can deprecate all cloneScopes since I do not see them used anywhere (even without this PR)